humble/general_humble

```diff
@@ -0,0 +1,45 @@
+from osrf/ros:humble-desktop-full
+
+# change the default shell to bash
+SHELL [ "/bin/bash", "-c" ]
+
+# install git
+RUN apt-get update && apt-get install -y \
+  net-tools \
+  iputils-ping\
+  git \
+  tmux \
+  python3-pip \
+  gazebo\
+  ros-humble-joy \
+  ros-humble-teleop-twist-joy \
+  ros-humble-teleop-twist-keyboard \
+  ros-humble-laser-proc \
+  ros-humble-ros-ign-bridge \
+  ros-humble-gazebo-ros-pkgs \
+  ros-humble-turtlebot3-msgs \
+  ros-humble-turtlebot3 
+#   ros-humble-xacro \
```

Comment:
Why are these commented? Do these not exist in humble?

Reply:
I just checked some of them are not supported by ROS2 but I will add some pacakges equivalent to these pkgs.